roxie/ccd/ccddali.cpp

```diff
@@ -805,7 +806,7 @@ class CRoxieDaliHelper : implements IRoxieDaliHelper, public CInterface
                 waitToConnect -= delay;
             }
         }
-        initializeStorageGroups(daliHelper->connected());
+        initializeStoragePlanes(daliHelper->connected(), false); // This can be called while queries are running - so is not thread safe
```

Comment:
not new, but it looks like connectToDali is called every time a query dll is loaded, unconditionally,
meaning there is excessive calling of loading/refreshing the configuration (may be slowing the init time down as well).
Should this only be called by when it 1st time and/or when it actually reconnects to Dali ?

Reply:
Created HPCC-33198 to look at that issue.